app/build.gradle

```diff
@@ -34,18 +40,25 @@ dependencies {
     provided 'org.projectlombok:lombok:1.12.6'
     provided 'org.roboguice:roboblender:3.0'
 
-    compile 'com.android.support:design:22.2.0'
-    compile 'com.android.support:support-v4:22.2.0'
-    compile 'com.android.support:appcompat-v7:22.2.0'
+    compile 'com.android.support:design:23.1.1'
+    compile 'com.android.support:support-v4:23.1.1'
+    compile 'com.android.support:appcompat-v7:23.1.1'
 
     compile 'com.google.guava:guava:18.0'
     compile 'org.roboguice:roboguice:3.0.1'
 
+    compile 'io.reactivex:rxjava:1.1.0'
+    compile 'io.reactivex:rxandroid:1.1.0'
+    compile 'com.trello:rxlifecycle:0.4.0'
+    compile 'com.trello:rxlifecycle-components:0.4.0'
+    compile 'com.jakewharton.rxbinding:rxbinding:0.3.0'
+
+    compile 'com.squareup.okhttp3:okhttp:3.0.1'
     compile 'org.apache.commons:commons-lang3:3.1'
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'org.codehaus.jackson:jackson-mapper-asl:1.9.13'
     compile 'com.github.chrisbanes.photoview:library:1.2.3'
-    compile 'com.loopeer.android.thirdparty:swipe-back-layout:1.0'
+    compile 'me.imid.swipebacklayout.lib:library:1.0.0'
```

Comment:
Replaced because the fork apparently included sample resources in the library...